sources/ClangSharp.PInvokeGenerator/PInvokeGenerator.VisitDecl.cs

```diff
@@ -2489,9 +2499,26 @@ void ForUnderlyingType(TypedefDecl typedefDecl, Type underlyingType)
                 {
                     ForPointeeType(typedefDecl, parentType: null, referenceType.PointeeType);
                 }
-                else if (underlyingType is TagType)
+                else if (underlyingType is TagType underlyingTagType)
                 {
-                    // Nothing to do for tag types
+                    // See if there's a potential typedef remapping we want to log
+                    if (_config.LogPotentialTypedefRemappings)
+                    {
+                        var typedefName = typedefDecl.UnderlyingDecl.Name;
+                        var possibleNamesToRemap = new string[] { "_" + typedefName, "_tag" + typedefName, "tag" + typedefName };
```

Comment:
Just looking at the Windows bindings I've done for my own project, what about `$"{typedefName}_tag"`, `$"{typedefName}_"`, and `$"{typedefName}__"`?